src/main/java/com/nokia/uwr/service/parser/BTSParserImpl.java

```diff
@@ -0,0 +1,48 @@
+package com.nokia.uwr.service.parser;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.nokia.uwr.scenario.bts.BTSDescription;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.springframework.stereotype.Service;
+
+import java.io.IOException;
+import java.util.List;
+
+/**
+ * Main implementation
+ *
+ * @author Piotr Stoklosa
+ */
+@Service
+public class BTSParserImpl implements BTSParser {
+
+    private static final Logger LOGGER = LogManager.getLogger(BTSParserImpl.class);
+
+    ObjectMapper mapper = new ObjectMapper();
+
+    /**
+     * Main implementation
+     *
+     * @author Piotr Stoklosa
+     */
+    @Override
+    public String parseBTSDescriptions(List<BTSDescription> descriptions) throws IllegalArgumentException {
+
+        if (descriptions == null) throw new IllegalArgumentException("descriptions is null");
```

Comment:
Create own exception instead of this one. It would tell more what happened

Reply:
I was thinking about that, but finally decided to leave IllegalArgumentException beacuse with throwing this I want only to tell that argument of this method shoud be legal (not null)